__tests__/materials/material.query.test.js

```diff
@@ -106,474 +98,439 @@ afterAll(async () => {
+                  value
+                }
+                simpleName {
+                  lang
+                  value
                 }
+                available
+                images {
+                  large
+                  medium
+                  small
+                  thumbnail
+                }
+              }
+              additionalPrice {
+                currency
+                value
               }
+              available
             }
-          `,
-        })
-        .catch(e => e);
-      expect(res.data.getAllMaterials).toBeDefined();
-      expect(res.data.getAllMaterials.items).toContainEqual({
-        __typename: 'Material',
-        name: [
-          { __typename: 'Language', lang: 'uk', value: 'Тест' },
-          { __typename: 'Language', lang: 'en', value: 'Test' },
-        ],
-        description: [
-          {
-            __typename: 'Language',
-            lang: 'uk',
-            value: 'Опис Тестового матеріальчика',
-          },
-          {
-            __typename: 'Language',
-            lang: 'en',
-            value: 'Description for Test Materialyy',
-          },
-        ],
-        purpose: 'test',
-        available: true,
-        additionalPrice: [
-          { __typename: 'CurrencySet', currency: 'UAH', value: 0 },
-          { __typename: 'CurrencySet', currency: 'USD', value: 0 },
-        ],
-        colors: [
-          {
-            __typename: 'Color',
-            code: 777,
-            name: [
-              { __typename: 'Language', lang: 'uk', value: 'Тестовий колір' },
-              { __typename: 'Language', lang: 'en', value: 'Test color' },
-            ],
-            images: {
-              __typename: 'ImageSet',
-              ...material.colors[0].images,
-            },
-            available: true,
-            simpleName: [
-              {
-                __typename: 'Language',
-                lang: 'uk',
-                value: 'проста назва кольору',
-              },
-              {
-                __typename: 'Language',
-                lang: 'en',
-                value: 'simple color name',
-              },
-            ],
-          },
-        ],
-      });
+          }
+        }
+      `,
     });
 
-    test('#2 Should receive one material', async () => {
-      const res = await client
-        .query({
-          query: gql`
-            query($id: ID!) {
-              getMaterialById(id: $id) {
-                ... on Material {
-                  name {
-                    lang
-                    value
-                  }
-                  description {
-                    lang
-                    value
-                  }
-                  purpose
+    const getAllMaterials = res.data.getAllMaterials;
+    expect(getAllMaterials).toBeDefined();
+    expect(getAllMaterials.items).toBeInstanceOf(Array);
+    expect(res.data.getAllMaterials.items).toContainEqual({
+      name: [
+        { lang: 'uk', value: 'Тест' },
```

Comment:
Please, don't hardcode values here, compare received values with the values from object `material` that is used in the `beforeAll` statements, it will simplify future modifications of tests

Reply:
fixed